### PR TITLE
Support for Quad Merlin

### PR DESCRIPTION
**Pull request by uellue**
Also rename `scan_size`-> `nav_shape` to follow dataset conventions.

## Contributor Checklist:

* [x] I have added or updated my entry in [the creators.json file](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/creators.json)
* [x] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
* [x] I have added/updated documentation for all user-facing changes
* [x] I have added/updated test cases

## Reviewer Checklist:

* [ ] `/azp run libertem.libertem-live-data` passed

### CODE REVIEW

File: src/libertem_live/detectors/merlin/data.py [hunk 1]
```diff
@@ -663,13 +677,54 @@ def stream(self, num_frames=None, chunk_size=11):
...
             while True:
                 with pool.get_result() as res_wrapped:
                     if pool.should_stop():
                         break
                     yield res_wrapped
+
+    def validate_get_sig_shape(self, hdr, sig_shape=None):
+        assembly_size = hdr.get('Assembly Size (NX1, 2X2)')
+        if assembly_size:
+            assembly_size = assembly_size.strip()
```
Comment (sk1p) [inline src/libertem_live/detectors/merlin/data.py:698]: Maybe like this, to also support the "gapped" assemblies?

```suggestion
            assembly_size = assembly_size.strip(" G")
```

Comment (uellue) [reply src/libertem_live/detectors/merlin/data.py]: Thx! :-) Let's see when we get actual data of that kind to test.

**Review by sk1p — APPROVED** [review]

**Pull request merged**